test_programs/execution_success/strings/src/main.nr

```diff
@@ -52,8 +62,8 @@ fn test_prints_array() {
 }
 
 fn failed_constraint(hex_as_field: Field) {
-    // TODO(#2116): Note that `println` will not work if a failed constraint can be 
-    // evaluated at compile time. 
+    // TODO(#2116): Note that `println` will not work if a failed constraint can be
+    // evaluated at compile time.
```

Comment:
This issue has since been marked solved so lets remove this TODO
```suggestion
```
Looking at the issue, I actually think it is the wrong issue number but the correct issue (which I can't find) should be fixed as well.